docker/development/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Dockerfile
+
+# Use node alpine as it's a small node image
+FROM node:alpine 
+RUN mkdir -p /app
+# Set /app as the working directory
+WORKDIR /app
+ENV NEXT_TELEMETRY_DISABLED=1
+# Copy package.json and package-lock.json
+# to the /app working directory
+COPY package*.json ./
+# Install dependencies in /app
+RUN npm install 
+# RUN npm i && npm cache clean --force
```

Comment:
Shall we just follow the template's command and use L14 instead? Think cleaning the cache saves a bit of memory

Reply:
That works

Reply:
```suggestion
RUN npm i && npm cache clean --force
```